src/pybind/CMakeLists.txt

```diff
@@ -18,6 +18,30 @@ foreach(
   list(APPEND NMODL_PYTHON_FILES_OUT ${PROJECT_BINARY_DIR}/nmodl/${file})
 endforeach()
 
+add_library(pyembed ${CMAKE_CURRENT_SOURCE_DIR}/pyembed.cpp)
+set_property(TARGET pyembed PROPERTY POSITION_INDEPENDENT_CODE ON)
+
+if(NOT LINK_AGAINST_PYTHON)
+  add_library(pywrapper SHARED ${CMAKE_CURRENT_SOURCE_DIR}/wrapper.cpp)
+else()
+  add_library(pywrapper ${CMAKE_CURRENT_SOURCE_DIR}/wrapper.cpp)
+  set_property(TARGET pywrapper PROPERTY POSITION_INDEPENDENT_CODE ON)
+  target_compile_definitions(pyembed PRIVATE STATIC_PYWRAPPER=1)
+endif()
+
+target_include_directories(pyembed PRIVATE ${PYBIND11_INCLUDE_DIR} ${PYTHON_INCLUDE_DIRS})
+target_include_directories(pywrapper PRIVATE ${pybind11_INCLUDE_DIRS} ${PYTHON_INCLUDE_DIRS})
```

Comment:
Ah, I got confused here but  see now this `PYBIND11_INCLUDE_DIR` and `pybind11_INCLUDE_DIRS` is very subtle!
Just FYI : when we build the wheel, there are no python lib/include available and hence PYBIND11_INCLUDE_DIR== pybind11_INCLUDE_DIRS.

Reply:
hum.. wait.. the `Python.h` won't be available when _building_ the wheel? that might become a problem. I believe pybind11 will need it to build embedded python related code!

Reply:
I have built pybind11 (and part of nmodl) before on manylinux and it's not a problem. (basically, with those linker flags, we ignore undefined symbols from libpython)

Reply:
Just to conclude on this. We discussed offline and we agree that the python headers are indeed available, so all good there - no change required.